bin/tag_rke2_release

```diff
@@ -4,28 +4,33 @@ set -e
 
 . libstd-ecm.sh
 
+DR_PREFIX=''
+MAIN_MINOR='23'
```

Comment:
I'm not sure we should be tightyly coupling to a version here.

Reply:
Even though you  can override that value with `-m` flag? 

An alternative that comes to my mind is to query rke2 master branch and look for that value in the `version.sh` wdyt?

Reply:
I guess I'd rather have the -m be required and be explicit.